avocado/core/test.py

```diff
@@ -940,7 +940,8 @@ def fail(self, message=None):  # pylint: disable=W0221
         """
```

Comment:
What do you think about changing the `fail` method to static just for consistency?

Reply:
Sure, that makes sense. Pylint seems to only issue ``no-self-use`` if the method has a ``return``, don't know if it is a feature or a bug... I think I can understand the reasoning behind why pylint does that (a method that does not modify anything would return something), but there are situations like this that can happen...

Reply:
Okay, I got it!